scripts/exregional_make_ics.sh

```diff
@@ -421,7 +421,7 @@ case "${EXTRN_MDL_NAME_ICS}" in
   vgfrc_from_climo=True
   minmax_vgfrc_from_climo=True
   lai_from_climo=True
-  tg3_from_soil=False
+  tg3_from_soil=True
```

Comment:
What happens here if do_surface_cycle is false?

Reply:
This is not related to surface cycle. It is what we should use in real-time.

Reply:
This is for chgres_cube option.

Reply:
So this isn't related to any of the other items included in this PR?

Reply:
This is option used in real-time run but didn't get commit several weeks ago when we work on surface cycling.